test/e2e/cypress/utils/photos.mock.ts

```diff
@@ -3,157 +3,189 @@ const URL_BRIDGE = Cypress.env('REACT_APP_STORJ_BRIDGE');
 const REACT_APP_PROXY = Cypress.env('REACT_APP_PROXY') ? Cypress.env('REACT_APP_PROXY') + '/' : '';
 
 export enum MOCK {
-    EMPTY = 'EMPTY',
-    PHOTO1 = 'PHOTO1',
-    PHOTO2 = 'PHOTO2',
-    MULTIPLE = 'MULTIPLE'
-};
+  EMPTY = 'EMPTY',
```

Comment:
Is there any change in the photos tests and mocks? I only see the formatting of the code. 
Just to make sure I don't miss anything.

Reply:
It's just code formatting